wallet/src/de/schildbach/wallet/ui/WalletActivity.java

```diff
@@ -320,7 +321,7 @@ private void initViewModel() {
         // the Join DashPay (evolution) button on the shortcuts bar.
         // If that is the only function that the platform required for, then we can
         // conditionally execute this code when a username hasn't been registered.
-        dashPayViewModel = ViewModelProviders.of(this).get(DashPayViewModel.class);
+        dashPayViewModel = new ViewModelProvider(this).get(DashPayViewModel.class);
```

Comment:
This old way of retrieving `ViewModel`s is deprecated in 'androidx.core:core:1.2.0', I applied the recommended one.